src/test/java/com/trycloud/stepDefinitions/LogIn_stepDef_TD.java

```diff
@@ -0,0 +1,81 @@
+package com.trycloud.stepDefinitions;
+
+import com.trycloud.pages.*;
+import com.trycloud.utility.*;
+import io.cucumber.java.en.*;
+import org.apache.logging.log4j.*;
+import org.junit.*;
+import org.openqa.selenium.interactions.*;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class LogIn_stepDef_TD {
+    Logger LOG = LogManager.getLogger();
+    LoginPage loginPage=new LoginPage();
+    Actions action=new Actions(Driver.getDriver());
+    @Given("the user is on the login page")
+    public void the_user_is_on_the_login_page() {
+        LOG.info("user goes to login page");
+    }
+    @When("the user enters valid {string} and {string}")
+    public void the_user_enters_valid_and(String userName, String password) {
+        loginPage.usernameBox.sendKeys(userName);
+        loginPage.passwordBox.sendKeys(password);
+
+    }
+
+    @Then("the user should be logged in successfully")
+    public void the_user_should_be_logged_in_successfully() {
```

Comment:
Empty method body?
Successful login results in user landing on dashboard page/ main page